Context.sublime-menu

```diff
@@ -1,7 +1,7 @@
 [
     { "caption": "-", "id": "adb" },
     { "command": "adb_filter_by_process_id", "caption": "ADB Filter by Process ID" },
-    { "command": "adb_filter_by_process_name", "caption": "ADB Filter by Process Name" },
+    { "command": "adb_filter_by_thread_id", "caption": "ADB Filter By Thread ID" },
```

Comment:
The "filter by process name" feature still makes sense. Could you just add a new command for "filter by thread id", instead of replacing an existing one? Also, could you update README.md for the new added command as well?